hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/BatchRestartRsAction.java

```diff
@@ -21,14 +21,19 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+
 import org.apache.hadoop.hbase.ServerName;
 import org.apache.hadoop.hbase.chaos.monkies.PolicyBasedChaosMonkey;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Restarts a ratio of the running regionservers at the same time
  */
 public class BatchRestartRsAction extends RestartActionBaseAction {
   float ratio; //ratio of regionservers to restart
+  private static final Logger LOG =
+          LoggerFactory.getLogger(BatchRestartRsAction.class);
```

Comment:
Should be intended at level 6.

Reply:
@HorizonNet I have made the necessary changes. Hopefully, changes made will address the indentation issue.

Reply:
Still not level 6. 6 is at the `v` of `private`.